src/Core/src/Handlers/ScrollView/ScrollViewHandler.Android.cs

```diff
@@ -147,7 +147,7 @@ public static void MapRequestScrollTo(IScrollViewHandler handler, IScrollView sc
 			var verticalOffsetDevice = (int)context.ToPixels(request.VerticalOffset);
 
 			handler.PlatformView.ScrollTo(horizontalOffsetDevice, verticalOffsetDevice,
-				request.Instant, () => handler.VirtualView.ScrollFinished());
+				request.Instant, () => scrollView?.ScrollFinished());
```

Comment:
can we leave the previous code but do a check to see if the handler is still connected?

```C#
if (this.Handler.IsConnected())
      handler.VirtualView.ScrollFinished()
```